src/components/Popup/types.ts

```diff
@@ -13,52 +13,77 @@ export type PopupProps = {
   /**
    * A ref to the element whose position will be used to anchor the Popup
    */
-  anchorRef: React.MutableRefObject<HTMLElement>
+  anchorRef: MutableRefObject<HTMLElement>
+
   /**
    * Duration of the animations in ms
    */
   animationDuration?: number
+
   /**
    * A function used to check if a click falls inside any element
    * that should not close the popup. Clicks inside the menu itself
    * are automatically considered inside
    */
   checkIfClickInside?: (target: EventTarget) => boolean
+
+  /**
+   * Children to render inside the Popup
+   */
   children: React.ReactChild
-  className?: string
+
   /**
    * Boolean representing whether the Popup is visible
    */
   isVisible: boolean
+
   /**
    * Show the header
    */
   showHeader?: boolean
+
   /**
    * Fired when a close event is dispatched, but the animation is not necessarily finished
    */
   onClose: () => void
+
   /**
    * Fired after the popup finishes closing
    */
   onAfterClose?: () => void
+
   /**
    * The position of the Popup relative to the trigger
    */
   position?: Position
+
   /**
    * A title displayed at the top of the Popup (only visible when the header is enabled)
    */
   title?: string
+
+  /**
+   * Class name to apply to the popup itself
+   */
+  className?: string
```

Comment:
The props are listed in alphabetical order, but if you think it makes more sense to keep `className` and `wrapperClassName` together we can keep it like this

Reply:
oh alphabetical works. didn't realize it was that. will change!